### PR TITLE
Feature autofetch comments

### PR DESCRIPTION
**Pull request by kalomak**
Fetch comments with 3s interval

## Review Checklist

- [x] All changes are described in the PR description
- [x] New feature and breaking changes have documentation
- [x] New endpoints have API tests
- [x] New frontend features have Enzyme tests
- [x] No unnecessary or debugging code
- [x] Questionable code is clearly marked with comments
- [x] Manually tested with latest Firefox, Safari and Chrome

### CODE REVIEW

**Comment by joonalillfors** [comment]
frontend tests?

File: client/src/feedbacker-components/comment-panel/comment-panel.js [hunk 1]
```diff
@@ -52,6 +52,27 @@ class CommentPanel extends React.Component {
...
+    if (this.interval) {
+      window.clearInterval(this.interval)
+      this.interval = null
+    }
+  }
+
+  async fetchComments() {
+    const comments = await apiCall('GET', '/comments')
+    this.props.dispatch(loadComments(comments))
+    this.scrollToBottom()
```
Comment (bqqbarbhg) [inline client/src/feedbacker-components/comment-panel/comment-panel.js]: Probably shouldn't scroll to bottom every 3 seconds?

Comment (bqqbarbhg) [reply client/src/feedbacker-components/comment-panel/comment-panel.js]: Need to have two `fetchComments()`, one for posting and one for polling

**Review by bqqbarbhg — CHANGES_REQUESTED** [review]

File: client/src/feedbacker-components/comment-panel/comment-panel.js (hunk 1 above)
Comment (kalomak) [reply client/src/feedbacker-components/comment-panel/comment-panel.js]: fixed

**Review by bqqbarbhg — APPROVED** [review]
Go for it

**Pull request merged**